### PR TITLE
Remove comments for resolved LWG issues

### PR DESCRIPTION
**Pull request by miscco**
This removes the comments for some of the resolved LWG issues.

#Addresses #1446

### CODE REVIEW

File: stl/inc/ranges [hunk 1]
```diff
@@ -1393,8 +1393,7 @@ namespace ranges {
 
             _NODISCARD friend constexpr difference_type operator-(const _Iterator& _Left,
                 const _Iterator& _Right) noexcept(noexcept(_Left._Current - _Right._Current)) /* strengthened */
-                requires sized_sentinel_for<iterator_t<_Base>, iterator_t<_Base>> { // constraints per LWG issue
-                                                                                    // unnumbered as of 2020-09-03
```
Comment (StephanTLavavej) [inline stl/inc/ranges:1397]: For the record, this was LWG-3483 `transform_view::`*`iterator`*'s difference is overconstrained.

**Review by StephanTLavavej — APPROVED** [review]

**Comment by CaseyCarter** [comment]
FWIW, I think I'd prefer to hold off reviewing this until we finish #1446 so we can continue to push any comment cleanups into this PR instead of having multiple such PRs.

**Comment by StephanTLavavej** [comment]
I believe I've pushed changes for all of the implemented-and-commented issues.

**Comment by miscco** [comment]
Awesome, thanks a lot ;)

**Review by cbezault — APPROVED** [review]

**Review by CaseyCarter — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
Thanks for cleaning up these comments (and implementing many of these resolutions in the ranges code!). :rocket:

**Pull request merged**